### ISSUE TITLE
Alerting: HoverCard component cannot be tabbed onto

### ISSUE THREAD

**Issue by JoaoSilvaGrafana** [grafana/grafana#64052]
**Steps to reproduce**:

1. Go to https://play.grafana.org/alerting/list
2. There is an information tooltip next to the "Search" box, try to tab onto it
<img width="689" alt="Screenshot 2023-03-02 at 16 11 28" src="https://user-images.githubusercontent.com/100691367/222485623-5bc8f25d-d961-4276-84bc-ba2a972abb19.png">

Similarly if you try to tab into a token on a rule description:
<img width="780" alt="Screenshot 2023-03-02 at 16 11 06" src="https://user-images.githubusercontent.com/100691367/222485543-9e80c8ae-9bc1-49d7-a95a-84876693e71e.png">

**Actual Result**:

It is not possible to tab into either of these components and therefore the parent component of `HoverCard` is not fully accessible

**Expected Result**

That using only the keyboard you can get the same information as if you could use a mouse.

**Relevant WCAG Criteria:** [2.1.1 - Keyboard](https://www.w3.org/WAI/WCAG21/quickref/#keyboard)

**Environment**:
- Grafana version: 9.4.1
- Data source type & version: n/a
- User OS & Browser: n/a
- Others:
